single_role_playbooks/jenkins.yml

```diff
@@ -1,5 +1,13 @@
 ---
-- hosts: jenkins
+- hosts: jenkins  
+  vars:
+    java_packages:
+      - java-1.8.0-openjdk
```

Comment:
This is rather old. According to the Jenkins docs:
-     OpenJDK JDK / JRE 8 - 64 bits
-     OpenJDK JDK / JRE 11 - 64 bits
are supported, so maybe updating to `java-11-openjdk` is Ok...